### ISSUE TITLE
Add also URI mappings for xml.xsd (and probably also ditaarch.xsd)

### ISSUE THREAD

**Issue by raducoravu** [dita-ot/dita-ot#2725]
If you look in the XML catalog for DITA 1.2:

`plugins\org.oasis-open.dita.v1_2\schema\catalog.xml`

it has both system ID mappings for "urn:oasis:names:tc:dita:xsd:xml.xsd" and similar system IDs and URI mappings like:

```xml
<uri name="xml.xsd" uri="xml.xsd"/>
<uri name="urn:oasis:names:tc:dita:xsd:xml.xsd" uri="xml.xsd"/>
<uri name="urn:oasis:names:tc:dita:xsd:xml.xsd:1.x" uri="xml.xsd"/>
<uri name="urn:oasis:names:tc:dita:xsd:xml.xsd:1.2" uri="xml.xsd"/>
```

but in the equivalent catalog.xml for the DITA 1.3 specs:

`plugins\org.oasis-open.dita.v1_3\schema\base\catalog.xml`

you only have the system ID mappings:

```xml
<system systemId="xml.xsd" uri="xsd/xml.xsd"/>
<system systemId="urn:oasis:names:tc:dita:xsd:xml.xsd" uri="xsd/xml.xsd"/>
<system systemId="urn:oasis:names:tc:dita:xsd:xml.xsd:1.x" uri="xsd/xml.xsd"/>
<system systemId="urn:oasis:names:tc:dita:xsd:xml.xsd:1.3" uri="xsd/xml.xsd"/>
```

Ideally the URI mappings would also be present there like:

```xml
<uri name="xml.xsd" uri="xsd/xml.xsd"/>
<uri name="urn:oasis:names:tc:dita:xsd:xml.xsd" uri="xsd/xml.xsd"/>
<uri name="urn:oasis:names:tc:dita:xsd:xml.xsd:1.x" uri="xsd/xml.xsd"/>
<uri name="urn:oasis:names:tc:dita:xsd:xml.xsd:1.3" uri="xsd/xml.xsd"/>
```

because there are XML Schema-based validators like Saxon EE which will use only URI mappings to resolve schema references.

**Comment by robander** [comment]
It looks like these have already been added to the DITA 1.3 errata - we should incorporate that into our next fix.

**Comment by robander** [comment]
Fixed with #2860
